app/index.html

```diff
@@ -10,9 +10,9 @@
 
     <link rel="stylesheet" href="css/main.css">
   </head>
-  <body ng-app='theshelf'>
-    <side-content></side-content>
-    <div ui-view></div>
+  <body class='Site' ng-app='theshelf'>
```

Comment:
And why call it `Site`? What happens if you have another one of these? What will you call it?